### PR TITLE
Update runtime programs location

### PR DESCRIPTION
**Pull request by rmoyard**
Change the runtime programs location and update two tests. One test had not enough parameters. The other tests is raising a warning because the number of parameters is updated during the computation.

### CODE REVIEW

File: tests/test_runtime.py [hunk 1]
```diff
@@ -325,7 +324,7 @@ def test_ansatz_qiskit(self, token, tol, shots):
             backend="ibmq_qasm_simulator",
             hamiltonian=hamiltonian,
             ansatz="EfficientSU2",
-            x0=[3.97507603, 3.00854038],
+            x0=[0.0, 0.0, 0.0, 0.0],
```
Comment (antalszava) [inline tests/test_runtime.py:327]: How come this was changed?

File: tests/test_runtime.py [hunk 2]
```diff
@@ -551,23 +550,29 @@ def test_too_many_param(self, token, tol, shots):
 
         def vqe_circuit(params):
             qml.RX(params[0], wires=0)
-            qml.RX(params[1], wires=0)
+            qml.RY(params[1], wires=0)
```
Comment (antalszava) [inline tests/test_runtime.py:553]: How come this was changed?

File: tests/test_runtime.py [hunk 3]
```diff
@@ -551,23 +550,29 @@ def test_too_many_param(self, token, tol, shots):
...
-        job = vqe_runner(
-            program_id=program_id,
-            backend="ibmq_qasm_simulator",
-            hamiltonian=hamiltonian,
-            ansatz=vqe_circuit,
-            x0=[3.97507603, 3.00854038, 3.55637849],
-            shots=shots,
-            optimizer_config={"maxiter": 10},
-            kwargs={"hub": "ibm-q-startup", "group": "ibm-q-startup", "project": "reservations"},
+        with pytest.warns(UserWarning) as record:
```
Comment (antalszava) [inline tests/test_runtime.py:561]: Worth adding a note on this to the PR description.

File: doc/requirements.txt [hunk 4]
```diff
@@ -1,4 +1,5 @@
 docutils==0.16
+mthree
```
Comment (antalszava) [inline doc/requirements.txt:2]: Minor: would this be required to build the docs?

**Review by antalszava — COMMENTED** [review]
Thank you @rmoyard! :slightly_smiling_face:

Could you confirm that running the suite with this PR in a fashion as the plugin test matrix does (`python -m pytest plugin_repo/tests -k 'not test_ibmq.py and not test_runtime.py' --tb=short`) works well and doesn't produce [an import error](https://github.com/PennyLaneAI/plugin-test-matrix/runs/5007267051?check_suite_focus=true#step:8:28)?

File: tests/test_runtime.py (hunk 1 above)
Comment (rmoyard) [reply tests/test_runtime.py:327]: We are using EfficientSu2 from the Qiskit library and it has 4 parameters when there are 2 qubits.

File: tests/test_runtime.py (hunk 2 above)
Comment (rmoyard) [reply tests/test_runtime.py:553]: It makes more sense to have two different operations here, anyway we are not interested in the actual result.

File: tests/test_runtime.py (hunk 1 above)
Comment (rmoyard) [reply tests/test_runtime.py:327]: We are using EfficientSu2 from the Qiskit library and it has 4 parameters when there are 2 qubits.

File: tests/test_runtime.py (hunk 3 above)
Comment (rmoyard) [reply tests/test_runtime.py:561]: Yes I am adding it!

File: doc/requirements.txt (hunk 4 above)
Comment (rmoyard) [reply doc/requirements.txt:2]: Yes it is necessary exactly.

**Review by antalszava — APPROVED** [review]
Thank you! :slightly_smiling_face:

**Pull request merged**